src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -35,9 +41,27 @@ public TimeSpan AutomaticRefreshInterval
                     throw LogHelper.LogExceptionMessage(new ArgumentOutOfRangeException(nameof(value), LogHelper.FormatInvariant(LogMessages.IDX10108, LogHelper.MarkAsNonPII(MinimumAutomaticRefreshInterval), LogHelper.MarkAsNonPII(value))));
 
                 _automaticRefreshInterval = value;
+                Interlocked.Exchange(ref _automaticRefreshIntervalInSeconds, value.TotalSeconds);
             }
         }
 
+        /// <summary>
+        /// Records the time this instance was created.
+        /// Used to determine if the automatic refresh or request refresh interval has passed.
+        /// The set is used for testing purposes.
+        /// </summary>
+        internal DateTimeOffset StartupTime { get; set; } = DateTimeOffset.UtcNow;
+
+        /// <summary>
+        /// Each time GetConfigurationAsync() results in a http request, this value is incremented.
+        /// </summary>
+        internal long NumberOfTimesAutomaticRefreshRequested { get; set; }
+
+        /// <summary>
+        /// Each time RequestRefresh() results in a http request, this value is incremented.
```

Comment:
However Visual Studio generates it:
```suggestion
        /// Each time <see cref="RequestRefresh" /> results in a http request, this value is incremented.
```